packages/api/src/models/plantTaskModel.js

```diff
@@ -31,20 +31,12 @@ class PlantTaskModel extends Model {
   static get jsonSchema() {
     return {
       type: 'object',
+      required: ['task_id', 'planting_management_plan_id'],
 
       properties: {
         task_id: { type: 'integer' },
-        space_depth_cm: {
-          type: 'float',
-        },
-        space_length_cm: {
-          type: 'float',
-        },
-        space_width_cm: {
-          type: 'float',
-        },
-        'rate_seeds/m2': {
-          type: 'float',
+        planting_management_plan_id: {
+          type: 'string',
```

Comment:
was there originally a plan to contain `space_depth`, `space_length`, etc. in planting_management_plan?